automated install/basic-install.sh

```diff
@@ -1124,13 +1124,13 @@ main() {
 		echo ":::     ${IPV6_ADDRESS}"
 		echo ":::"
 		echo "::: If you set a new IP address, you should restart the Pi."
+    echo "::: View the web interface at http://pi.hole/admin or http://${IPV4_ADDRESS%/*}/admin"
```

Comment:
Ah, that's where that was. And imported the codeStyles, `.editconfig` seems kind of on and off with it's style settings.